src/test/resources/application-test.yml

```diff
@@ -37,4 +37,8 @@ hmpps.sqs:
       subscribeTopicId: hmppseventtopic
   topics:
     hmppseventtopic:
-      arn: arn:aws:sns:eu-west-2:000000000000:${random.uuid}
\ No newline at end of file
+      arn: arn:aws:sns:eu-west-2:000000000000:${random.uuid}
+
+service:
+  active:
+    prisons: ${SERVICE_ACTIVE_PRISONS:***}
```

Comment:
Perhaps change this config and the test to be "LEI,LPI,MDI" (Leeds, Moorlands and Liverpool) to confirm that the info returns an array of active agencies i.e. `"activeAgencies":["LEI","LPI","MDI"]` as we're at the stage of enabling individual prisons rather than all prisons?